### PR TITLE
Update test.js

### PR DESCRIPTION
**Pull request by NguyenHieu71**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1 +1,2 @@
-console.log("afsfasfaf");
\ No newline at end of file
+console.log("afsfasfaf");
+fuck u
```
Comment (NguyenTrongCuong) [inline test.js:2]: ? very bad

**Comment by NguyenTrongCuong** [comment]
fix right now bitch :>

File: test.js (hunk 1 above)
Comment (NguyenHieu71) [reply test.js:2]: what do u mean ?

Comment (NguyenTrongCuong) [reply test.js:2]: it means fuck u, go fix it right now

**Review by NguyenTrongCuong — APPROVED** [review]
okay u're in

**Pull request merged**